quickwit-metastore/src/lib.rs

```diff
@@ -52,7 +52,7 @@ mod metastore_resolver;
 pub mod postgresql;
 
 pub use error::{MetastoreError, MetastoreResolverError, MetastoreResult};
-pub use metastore::file_backed_metastore::FileBackedMetastore;
+pub use metastore::file_backed_metastore::{FileBackedMetastore, Index};
```

Comment:
I don't think we should expose it as quickwit_metastore::Index...
At least under this name. This will make things very confusing.

This Index is an internal detail of the file backed metastore. 
If it wasn't for regression tests, it should only be visible in the file_backed_metastore module.

Reply:
Can I make `file_backed_metastore` public without making `metastore` public, though?